api/src/main/java/io/druid/js/JavaScriptConfig.java

```diff
@@ -73,8 +73,8 @@ public String toString()
            '}';
   }
 
-  public static JavaScriptConfig getDefault()
+  public static JavaScriptConfig getEnabled()
```

Comment:
I did a double-take on this, thinking it was a method getting whether or not something was enabled. `getEnabledInstance()` or `enabledInstance()` would probably be clearer.